examples/Detoxify.ipynb

```diff
@@ -198,6 +204,62 @@
     "tmarco = TMaRCo()"
    ]
   },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "This will initialize `TMaRCo` using the default models, taken from HuggingFace.\n",
+    "<div class=\"alert alert-info\">\n",
+    "To use local models with TMaRCo, we need to have them in a local storage, accessible to TMaRCo, initialize separately, and pass them to TMaRCo's constructor.\n",
```

Comment:
Can we update the wording here? Not sure I understand what this is requiring. Is it:

```
Prequisites:
1) The models in local storage
2) The storage needs to be accessible to TMaRCO
3) The models have been initialized separately
```

If so, we could say:
"...we need to have the pre-initialized models in a local storage that is accessible to TMaRCo"